server/src/client_api.rs

```diff
@@ -129,7 +129,7 @@ pub async fn get_feature(
         .ok_or(EdgeError::AuthorizationDenied)?;
 
     let filter_set = FeatureFilterSet::from(Box::new(name_match_filter(feature_name.clone())))
-        .with_filter(project_filter(&edge_token));
+        .with_filter(project_filter(&validated_token));
```

Comment:
I don't know, but I suspect this line would suffer from the same problem?

Reply:
Checking it out at the moment

Reply:
Can confirm that this does fix another bug, but there's more issues with it, so I want to add a test in a new PR.

Reply:
I think you're correct, this has been a lesson in misplaced assumptions. Thank you for fixing it @thomasheartman :clap:

Reply:
The other issue floating around is here: https://github.com/Unleash/unleash-edge/issues/276